src/test/java/com/example/spring_boot_memo/SecurityTest.java

```diff
@@ -0,0 +1,74 @@
+package com.example.spring_boot_memo;
+
+
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestBuilders.logout;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import java.security.NoSuchAlgorithmException;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        classes = SpringBootMemoApplication.class)
+@AutoConfigureMockMvc
+public class SecurityTest {
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    PasswordEncoder passwordEncoder;
+
+    @Test
+    @WithMockUser
+    @DisplayName("認証成功後にアクセスページにログインアクセスできるか確認")
+    public void whenGetCustomers_thenStatus200() throws Exception {
+        mvc.perform(get("/menu/menu"))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    @WithMockUser
+    @DisplayName("認証後にログアウトのエンドポイントへリクエストすると、/logoutへリダイレクトする")
```

Comment:
```suggestion
    @DisplayName("認証後にログアウトのエンドポイントへリクエストすると、/logout?logoutへリダイレクトする")
```

LocationのURL、パスしか表現していないように見えます。
ドメイン含めたＵＲＬにする必要はありませんか？

環境の場合
https://{your-domain}/logout?logout

Reply:
tesuto

Reply:
ドメインを含む方法があるか調べてみてたのですが、見つからなかったです。
実装方法がわからん or 時間がかかりそうです。